test/hardwareWallet.js

```diff
@@ -0,0 +1,59 @@
+const { assert } = require('chai');
+let bcoin;
+
+if (process.env.BROWSER) bcoin = require('bcoin/lib/bcoin-browser');
+else bcoin = require('bcoin');
+
+const { HDPublicKey } = bcoin;
```

Comment:
Could this be destructured in the require statement? Not a big deal when we're not bundling, but as a convention it helps avoid webpack pulling in all of bcoin.

Reply:
This is in a test so it shouldn't matter?

Reply:
Yeah, that's what I meant about the bundling, just a comment in terms of the convention for people using this as a reference but not thinking about the difference. Basically just a nit.